pkg/bugzilla/bugzilla.go

```diff
@@ -56,27 +55,33 @@ var (
 // lgtm'd by the bug's QA Contect
 func (c *Verifier) VerifyBugs(bugs []int) []error {
 	bzPRs, errs := getPRs(bugs, c.bzClient)
-	for _, bzp := range bzPRs {
-		bug, err := c.bzClient.GetBug(bzp.bugID)
+	for bugID, extPRs := range bzPRs {
+		bug, err := c.bzClient.GetBug(bugID)
 		if err != nil {
-			errs = append(errs, fmt.Errorf("Unable to get bugzilla number %d: %v", bzp.bugID, err))
+			errs = append(errs, fmt.Errorf("Unable to get bugzilla number %d: %v", bugID, err))
 			continue
 		}
 		if bug.Status != "ON_QA" {
 			continue
 		}
-		comments, err := c.ghClient.ListIssueComments(bzp.org, bzp.repo, bzp.prNum)
-		if err != nil {
-			errs = append(errs, fmt.Errorf("Unable to get comments for github pull %s/%s#%d: %v", bzp.org, bzp.repo, bzp.prNum, err))
-			continue
-		}
-		reviews, err := c.ghClient.ListReviews(bzp.org, bzp.repo, bzp.prNum)
-		if err != nil {
-			errs = append(errs, fmt.Errorf("Unable to get reviews for github pull %s/%s#%d: %v", bzp.org, bzp.repo, bzp.prNum, err))
-			continue
+		// TODO: use this list to comment on bugzilla bugs with reason for not auto-verifying a PR
```

Comment:
Why leave this TODO?

Reply:
I was planning to do all the bugzilla commenting work as part of a separate PR after this one is merged. This `TODO` was put there mostly to indicate why I changed this to an array when I could have left it as a boolean as before.